frame/assets/src/types.rs

```diff
@@ -32,7 +32,7 @@ pub(super) type AssetAccountOf<T, I> =
 /// AssetStatus holds the current state of the asset. It could either be Live and available for use,
 /// or in a Destroying state.
 #[derive(Clone, Encode, Decode, Eq, PartialEq, RuntimeDebug, MaxEncodedLen, TypeInfo)]
-pub(super) enum AssetStatus {
+pub enum AssetStatus {
```

Comment:
why did you remove the `(super)`?

Reply:
I had to remove the (super) because of the runtime migration